src/main/java/thePackmaster/cards/goddessofexplosionspack/ExplosiveForm.java

```diff
@@ -0,0 +1,27 @@
+package thePackmaster.cards.goddessofexplosionspack;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.powers.goddessofexplosionspack.ExplosiveFormPower;
+import thePackmaster.util.Wiz;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class ExplosiveForm extends AbstractGoddessOfExplosionsCard {
+    public final static String ID = makeID("ExplosiveForm");
+
+    public ExplosiveForm() {
+        super(ID, 3, CardType.POWER, CardRarity.RARE, CardTarget.SELF);
```

Comment:
Having both rare powers be 3 costs might not be a great idea. Would probably be a good call to reduce one of the costs by 1 and nerf something else about it if needed.